db/migrate/20170908160100_add_ip_address_and_user_agent_id_to_event_logs.rb

```diff
@@ -0,0 +1,21 @@
+require 'lhm'
+
+class AddIpAddressAndUserAgentIdToEventLogs < ActiveRecord::Migration
+  def self.up
+    Lhm.cleanup(:run)
+    Lhm.change_table :event_logs do |m|
+      m.add_column :ip_address, "BIGINT"
+      m.add_column :user_agent_id, "INT"
+      m.ddl("ALTER TABLE %s ADD CONSTRAINT event_logs_user_agent_id_fk FOREIGN KEY (user_agent_id) REFERENCES user_agents(id)" % m.name)
+    end
+  end
+
+  def self.down
+    remove_foreign_key :event_logs, name: :event_logs_user_agent_id_fk
+    Lhm.change_table :event_logs do |m|
+      Lhm.cleanup(:run)
```

Comment:
Does the cleanup need to happen outside the change_table block, or inside?  In the `up` it's outside, but in the `down` it's inside.  Probably best for it to be consistent.

Reply:
I don't think it matters but I'll put them both outside for consistency